styleguide/markdown-syntax-style.md

```diff
@@ -1,167 +1,102 @@
-* Be mindful of **hyphens**! Hyphens for compound adjectives, no hyphens in verbs: 'end-user documentation' vs. 'for the end user'.
-
-### Headings
-
-No multi-sentence headings or full stops at the end of a sentence in page or sentence headings.
-Be consistent about heading levels - H1, H2, H3 - no jumping from H1 to H3 or H2 to H4.
+| Links                | Relative paths to other docs, full paths to anything else | Always full paths, check for HTTPS and that you're not using `localhost` | Always full paths, check for HTTPS and that you’re not using `localhost` |
 
 ## Code and filenames
 
-This section deals with adding and properly formatting code in the documentation + naming files, pages, and their derivative URLs.
+This section deals with adding and properly formatting code in the documentation, as well as naming files, pages, and their derivative URLs.
 
 ### Code formatting
 
-We use the GitHub flavor of markdown for [formatting code](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+The Buildkite docs uses the GitHub flavor of Markdown for [formatting code](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+
+For code or filenames as a part of a sentence, use a backtick "\`\" before and after the words that need to be marked as code. For example:
 
-For code or file names as a part of a sentence, use "\`\" before and after the word(s) that need(s) to be marked as code: Each command step can run either a shell command like `npm install`, or an executable file or script like `build.sh`.
-In markdown this sentence looks like this: " Each command step can run either a shell command like \`npm install\`, or an executable file or script like \`build.sh\`. "
+```md
+Each command step can run either a shell command like `npm install`, or an executable file or script like `build.sh`.
+```
 
-Do not use code fragments in page headings or section headings.
+> [!NOTE]
+> Do not use code fragments in page headings or section headings.
 
 ### Code blocks
 
-A code example longer than a couple of words that isn’t part of a sentence/a multi-line code sample needs to be formatted as a code block according to the [GitHub markdown flavor](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
-To add a code block, indent it using four (4) spaces or use 3 backticks (\`\`\`) before and after the code block.
+A code example longer than a couple of words that isn’t part of a sentence/a multi-line code sample needs to be formatted as a code block according to the [GitHub Markdown flavor](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+To add a code block, indent it using four (4) spaces or use three (3) backticks (\`\`\`) before and after the code block.
```

Comment:
Could we omit the reference to 4 spaces and just stick with one style—3 backticks?

Reply:
Good, do so.